### ISSUE TITLE
Merge projects?

### ISSUE THREAD

**Issue by skjerns** [cleemesser/python-edf#2]
Welcom to GitHub!

I'm one of the contributors of `pyedflib` and just recently found out that it was originally a fork of your project `python-edf`. However, much bug-fixing and adaptations have been put into `pyedflib`, and it would be more efficient to combine our efforts.

So I though to propose the idea to merge the two projects once again. What do you think? I don't see any obvious discrepancies between the goals and roadmaps of the two projects

**Comment by cleemesser** [comment]
Hi, just noticed that I had github notifications.
I am completely open to a merge. I tend to recommend your version for general users already. The only case where I use my own package is when I have some research code that relies upon it already written.

In terms of the future, I would like to ultimately move away from using the current C library, edflib. I would like to use a caching system with mmap for more efficient access as currently the main way the library is used is to load the entire edf file into memory then do work. This is awkward with large edf files.

**Comment by skjerns** [comment]
Yes, moving away from edflib might have some benefits and make life easier, but would be quite a lot of work and room for error. Additionally it might be substantially slower, at least the Python-EDF port by teuniz is much slower then his C-edflib. Let me know if you get something going!
